genesis-tools/genesis-csv-to-json/res/test_accounts.csv

```diff
@@ -0,0 +1,3 @@
+account_id,regular_pks,privileged_pks,foundation_pks,full_pks,amount,is_treasury,validator_stake,validator_key,peer_info,smart_contract,lockup,vesting_start,vesting_end,vesting_cliff
```

Comment:
what's the difference between regular vs full?
may be let's add a readme file with the explanation for columns.

Reply:
Anything but full is restricted to a specific set of contract methods. Regular is the one used by the user on the regular basis, e.g. for staking. Privileged is there to create/revoke regular keys. Foundation keys can stop vesting schedule. Full keys (if present. For anyone with vesting or lockup they should not be present) give full control, e.g. deployment of contracts.

I will add extensive documentation on all that. This PR is not final, I am just trying to submit it in parts because people are making changes to primitives and crypto and I don't want to be merging with it all the time.